### ISSUE TITLE
Source code Editor not readable for screenreader

### ISSUE THREAD

**Issue by snape84** [contao/core#8390]
With screenreader i am not able to read the text in the input field of the source code editor (Quelltext-Editor) in Contao. 
If I move thorugh the text with the arrow key, it can be seen on the screen that i am realy moving, but my Screenreader doesnt read anything to me. 
Because it looks for me like a normal text are field, I have o idea where exactly is the problem. Just let me know if I can provide more informations.

**Comment by aschempp** [comment]
Just as a quick solution, you can turn off the code editor in your user profile.

**Comment by snape84** [comment]
Thanks for the quick answer. The normal Rich text editor is accessible as it seems. The problem only occurs when i use the element type html.

**Comment by aschempp** [comment]
I know, and it's certainly something we should fix (if possible). Just wanted to point out that you can enable/disable each editor in your user profile. The code editor is more like a visual tool, so it might be ok to disable it for screenreader users.

**Comment by leofeyer** [comment]
This seems to be a [known issue](https://github.com/ajaxorg/ace/issues/2164). Until the ACE developers fix it, you should disable the code editor in your user settings as suggested by @aschempp.
